README.md

```diff
@@ -1,5 +1,17 @@
 # fourkitchens-karaoke
 
-[![CircleCI](https://circleci.com/gh/generalredneck/fourkitchens-karaoke.svg?style=shield)](https://circleci.com/gh/generalredneck/fourkitchens-karaoke)
+[![CircleCI](https://circleci.com/gh/fourkitchens/fourkitchens-karaoke.svg?style=shield)](https://circleci.com/gh/generalredneck/fourkitchens-karaoke)
 [![Dashboard fourkitchens-karaoke](https://img.shields.io/badge/dashboard-fourkitchens_karaoke-yellow.svg)](https://dashboard.pantheon.io/sites/ab2c6c8d-facb-4664-90fb-582a0d236de2#dev/code)
-[![Dev Site fourkitchens-karaoke](https://img.shields.io/badge/site-fourkitchens_karaoke-blue.svg)](http://dev-fourkitchens-karaoke.pantheonsite.io/)
\ No newline at end of file
+[![Dev Site fourkitchens-karaoke](https://img.shields.io/badge/site-fourkitchens_karaoke-blue.svg)](http://dev-fourkitchens-karaoke.pantheonsite.io/)
+
+
+## Local Development Requirements
+- [Lando](https://docs.devwithlando.io/installation/installing.html)
+- Node.js > 14.16.1 ([NVM](https://github.com/creationix/nvm) is recommended managing Node)
+- [Composer](https://getcomposer.org/)
+- [Terminus](https://pantheon.io/docs/terminus/install)
+
+## Local Setup
+
+- `npm install`
+- `npm run setup`
```

Comment:
Might consider getting the readme example from https://github.com/fourkitchens/Readme-example

Reply:
I looked at that one I'll add more to this.